src/main/java/com/gachigang/ontherun/config/security/BaseSecurityConfig.java

```diff
@@ -14,6 +17,10 @@ public class BaseSecurityConfig {
 
     @Bean
     public SecurityFilterChain securityFilterChain (HttpSecurity http) throws Exception {
+        DelegatingServerLogoutHandler logoutHandler = new DelegatingServerLogoutHandler(
```

Comment:
If you are able to explain to me what this class uses for and why did you choose this approach -- then no question, but if you just copied it from somewhere -- please try to find out any better solutions that are not so tiny bounded with spring classes  ( We want to provide our implementation of logout handler) -- and also why it is committed in the scope of prepopulating DB?